imgui not downloading with trunk checkout - imgui.h missing error

I'm using the anura-git package from the AUR in archlinux. It just checks out the 'trunk' branch from git and does 'make'
I get:

`Building: src/kre/imgui_impl_sdl_gl3.cpp
src/kre/imgui_impl_sdl_gl3.cpp:13:19: fatal error: imgui.h: No such file or directory
 #include "imgui.h"
                   ^
compilation terminated.
make: *** [Makefile:223: build/kre/imgui_impl_sdl_gl3.o] Error 1`
I don't know too much about how to use git, but it looks like the imgui folder isn't coming down with the rest. Am I doing something wrong?

--- a/.gitmodules
+++ b/.gitmodules
@@ -1,3 +1,3 @@
 [submodule "imgui"]
 	path = imgui
-	url = git@github.com:sweetkristas/imgui.git
+	url = https://github.com/sweetkristas/imgui

